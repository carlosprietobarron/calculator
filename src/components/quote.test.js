import RRender from 'react-test-renderer';
import Quote from './quote';

it('renders the Quote Page correctly', () => {
  const tree = RRender.create(<Quote />).toJSON();

  expect(tree).toMatchSnapshot();
});
