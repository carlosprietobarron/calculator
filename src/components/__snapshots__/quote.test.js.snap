// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Quote Page correctly 1`] = `
<div>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
  </p>
</div>
`;
